cmd/flow-rosetta-server/README.md

```diff
@@ -10,11 +10,12 @@ This allows the Rosetta API to access state remotely, or locally by running the
 ## Usage
 
 ```sh
-Usage of flow-dps-rosetta:
-  -a, --api string     host URL for GRPC API endpoint (default "127.0.0.1:5005")
-  -c, --chain string   chain ID for Flow network core contracts (default "flow-testnet")
-  -l, --log string     log output level (default "info")
-  -p, --port uint16    port to serve Rosetta API on (default 8080)
+Usage of ./flow-rosetta-server:
+  -a, --api string              host URL for GRPC API endpoint (default "127.0.0.1:5005")
+  -c, --chain string            chain ID for Flow network core contracts (default "flow-testnet")
+  -l, --level string            log output level (default "info")
+  -p, --port uint16             port to host Rosetta API on (default 8080)
+  -t, --transaction-limit int   maximum amount of transactions to include in a block response (default 10)
```

Comment:
That's way too low of a default. Please go for `200`. Also, don't forget that we have explicit explanations for each command line parameter, so I would prefer to just have `--transactions` over the more explicit `--transaction-limit`, because it's shorter to type and people looking to use them will see the help message anyway.